pkg/kube/pod_controller.go

```diff
@@ -54,13 +54,14 @@ type PodController interface {
 
 // podControllerProcessor aids in unit testing
 type podControllerProcessor interface {
-	createPod(ctx context.Context, cli kubernetes.Interface, options *PodOptions) (*corev1.Pod, error)
-	waitForPodReady(ctx context.Context, namespace, podName string) error
-	waitForPodCompletion(ctx context.Context, namespace, podName string) error
-	deletePod(ctx context.Context, namespace string, podName string, opts metav1.DeleteOptions) error
+	CreatePod(ctx context.Context, cli kubernetes.Interface, options *PodOptions) (*corev1.Pod, error)
+	WaitForPodReadyPCP(ctx context.Context, namespace, podName string) error
+	WaitForPodCompletionPCP(ctx context.Context, namespace, podName string) error
+	DeletePod(ctx context.Context, namespace string, podName string, opts metav1.DeleteOptions) error
 }
 
 // podController specifies Kubernetes Client and PodOptions needed for creating Pod
+// implements podControllerProcessor interface
```

Comment:
```suggestion
// podController specifies Kubernetes Client and PodOptions needed for creating
// a Pod. It implements the podControllerProcessor interface.
```
[nit] I learned recently that it's recommended to end comments with a period. Fixing this in the entire codebase will be taken up shortly.